tests/factories/requests_.py

```diff
@@ -0,0 +1,50 @@
+import json
+from io import BytesIO
+
+import requests
+from factory import Factory, Faker
+
+
+class Response(Factory):
```

Comment:
We create `request.Response` objects all over the place, and it's quite hard to do, as you aren't supposed to (only `requests`). So this is helpful here, but I think there's probably similar code elsewhere.

Unfortunately for us `factoryboy` doesn't make this easy, so the code is _way_ more long winded than an equivalent custom class would be.

Reply:
I've sent a PR to simplify this factory: https://github.com/hypothesis/lms/pull/1946